src/settings-ui/Settings.UI/Views/ShellPage.xaml.cs

```diff
@@ -326,6 +327,19 @@ private void NavigationView_SelectionChanged(NavigationView sender, NavigationVi
             }
         }
 
+        private void ExitPTItem_Tapped(object sender, Microsoft.UI.Xaml.Input.TappedRoutedEventArgs e)
+        {
+            const string ptTrayIconWindowClass = "PToyTrayIconWindow"; // Defined in runner/tray_icon.h
+            const UIntPtr ID_EXIT_MENU_COMMAND = 40001;                // Generated resource from runner/runner.base.rc
+
+            // Exit the XAML application
+            Application.Current.Exit();
```

Comment:
Should there be a dialog here first?

Reply:
I don't think the current Powertoys provides a dialog  

I am not too familiar with the code, but wouldn't there be another place that would handle the `Exit` command for the tray icon?  Should it call the same handler rather than exiting explicitly here?

Reply:
I do ultimately call that handler. `Application.Current.Exit()` closes the XAML window but leaves the tray process running in the background as it is a separate process. The following lines are:
```cs
// Invoke the exit command from the tray icon
IntPtr hWnd = NativeMethods.FindWindow(ptTrayIconWindowClass, ptTrayIconWindowClass);
NativeMethods.SendMessage(hWnd, NativeMethods.WM_COMMAND, ID_EXIT_MENU_COMMAND, 0);
```
which gets the tray icon pointer as `hWnd` and sends it the `ID_EXIT_MENU_COMMAND` which is generated from the resources file and corresponds to clicking the exit button in the right click menu.

Ultimately this does call the same handler, but the `Application.Current.Exit()` call is necessary to close the UI window.

Reply:
As this calls the normal exit menu command, any dialog should be done through that (which it currently is not) and would thus be outside of the scope of this PR I think.